src/app/services/match.service.ts

```diff
@@ -32,4 +32,10 @@ export class MatchService {
       });
       return homeWinSets + " - " + awayWinSets
   }
+
+  setMatchReferee(matchId: number, refereeId: number) {
+    console.log(matchId, refereeId)
+    return this.http.put(`/matches/${matchId}/referee`, {id: refereeId})
+    .subscribe(data => {console.log(data);});
```

Comment:
Well, I will remove the subscribe part, since the only thing it is doing is logging. But I will have that in mind in the future :)
And I am very confused about when to use semi-colons and not 😕

Reply:
I guess at some point we ought to think about error handling here though. ... But why start now :rocket: